src/components/tool/guide/Instructions/StartInstructions.js

```diff
@@ -59,8 +60,26 @@ export function JupyterHubStartInstructions({jobProfile}) {
             From the JupyterHub home page, click <Code>Start My Server</Code> to schedule a job.
         </InstructionText>
         <InstructionText>
-            From the list of available options, select the one containing <Code>Lyra - {jobProfile}</Code> and click the
-            start button.
+            From the list of available options, select the one containing <Code>Lyra - {jobProfile}</Code> and
+            click the <Code>Start</Code> button.
+        </InstructionText>
+        <InstructionHeading>Waiting Screen</InstructionHeading>
+        <InstructionText>
+            You will now enter a waiting screen and see the following message:
+            <br/>
+            <Code>Your server is starting up.</Code><br/>
+            <Code>You will be redirected automatically when it's ready for you</Code>
+        </InstructionText>
+        <InstructionText>
+            Wait for the screen to change when the server is ready for use.
+        </InstructionText>
+        <InstructionText>
+            It generally takes around 2-3 minutes for the server to start up, but it can take up to 5-10 minutes,
+            depending on the CPU/GPU units you requested. If it is taking too long please{' '}
+            <Link color='teal.500' href='https://eresearchqut.atlassian.net/servicedesk/customer/portals' isExternal>
+                submit a ticket at portal.
+                <ExternalLinkIcon mx='2px' />
+            </Link>
```

Comment:
```suggestion
            It may take up to 10 minutes for your job to be queued and for the
            server to start. If it fails to start, please raise an incident in the{" "}
            <Link
              color="teal.500"
              href="https://eresearchqut.atlassian.net/servicedesk/customer/portals"
              isExternal
            >
              eResearch help centre
              <ExternalLinkIcon mx="2px" />
            </Link>
            .
```

Reply:
Updated.